python/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
Intriguing. Yeah dev container is currently pinned to 3.9 because that's what `docker-geosupport` image is based on, so for consistency's sake both build images are using 3.9 as well. Would be happy to bump if we want though

Reply:
down to stick with 3.9 for now